### PR TITLE
add recept email after payment complete

### PR DESCRIPTION
**Pull request by btkostner**
This adds mandrill service for sending payment emails.

Some `TODO` tasks that are not blockers, but need to get done:
```
TODO: we need to save parsed appstream data for more human friendly names
TODO: save icon path or data for email receipts
```

### CODE REVIEW

**Comment by btkostner** [comment]
Attaching this PR, as it's needed for emails to work with AppCenter elementary/AppCenter#239

File: config.example.js [hunk 1]
```diff
@@ -37,6 +37,13 @@ module.exports.stripe = {
   public: 'pk_test_c32j0239j9tg8902j3f90238'
 }
 
+// https://mandrillapp.com/settings
+module.exports.mandrill = {
+  key: 'vw4861vwae4bv-4v8e1w2z',
```
Comment (lewisgoddard) [inline config.example.js]: Is this key okay to make public, or is it their example?

File: src/houston/controller/api/payment.js [hunk 2]
```diff
@@ -79,69 +80,122 @@ route.get('/:project', async (ctx) => {
...
+const sendReceipt = async (project: Project, email: string, amount: number) => {
+  log.debug('Sending payment email')
+
+  try {
+    await postReceipt(project, email, amount)
+
+    log.debug('Payment email sent')
+  } catch (err) {
+    log.error('Unable to send payment email')
+    log.error(err)
```
Comment (lewisgoddard) [inline src/houston/controller/api/payment.js]: Do we want to log the email address here?

File: test/fixtures/config.js [hunk 3]
```diff
@@ -27,6 +27,11 @@ module.exports.stripe = {
   public: 'pk_test_c32j0239j9tg8902j3f90238'
 }
 
+module.exports.mandrill = {
+  key: '489we1f5v156a-f78zxg9e',
```
Comment (lewisgoddard) [inline test/fixtures/config.js]: Is this key okay to make public?

File: config.example.js (hunk 1 above)
Comment (btkostner) [inline config.example.js]: it's an example

File: test/fixtures/config.js (hunk 3 above)
Comment (btkostner) [inline test/fixtures/config.js]: this is also an example

File: src/houston/controller/api/payment.js (hunk 2 above)
Comment (btkostner) [inline src/houston/controller/api/payment.js]: Possibly. I have the email send when we report it to stripe.

**Pull request merged**